backend/crawler/crawler.py

```diff
@@ -6,40 +6,185 @@
 checks cancel_flag of it.
 If cancel_flag is True, it stops current job and set status to 'canceled'.
 """
+import os
+
+dir_path = os.path.dirname(__file__)
+driver_path = os.path.join(dir_path, '../../chromedriver')
```

Comment:
I think it is better to get `driver_path` based on `BASE_DIR` of `settings.py`, not based on this file.
https://github.com/SWPP/ttrs-backend/blob/8646f2d124ed11e9c91437b9b63b35799fd2d3e6/backend/backend/settings.py#L15-L16

like this,
```python3
from django.conf import settings

driver_path = os.path.join(settings.BASE_DIR, ...)
```